packages/database/src/DatabaseModel.js

```diff
@@ -10,11 +10,6 @@ export class DatabaseModel {
 
   constructor(database) {
     this.database = database;
-    // Add change handler to database if defined, and this is the singleton instance of the model
```

Comment:
We had two patterns achieving the same thing. I've removed `onChange` in favour of `notifiers`